### PR TITLE
Release for tektoncd-pipeline-operator

### PR DESCRIPTION
**Pull request by arilivigni**
- config
 - jobs

### CODE REVIEW

File: ci-operator/config/openshift/tektoncd-pipeline-operator/openshift-tektoncd-pipeline-operator-release-next.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  project_image:
+    dockerfile_path: openshift-ci/Dockerfile.tools
+
+binary_build_commands: make build
+
+tag_specification:
+  name: "4.0"
+  namespace: ocp
+
+tests:
```
[the remark this answers is not in the document]
Comment (vdemeester) [reply ci-operator/config/openshift/tektoncd-pipeline-operator/openshift-tektoncd-pipeline-operator-release-next.yaml]: agreed with @nikhil-thomas, let's start simple, and iterate from there

File: ci-operator/config/openshift/tektoncd-pipeline/OWNERS [hunk 2]
```diff
@@ -3,6 +3,25 @@
 # See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
 
 approvers:
-- build-pipeline-approvers
```
Comment (vdemeester) [inline ci-operator/config/openshift/tektoncd-pipeline/OWNERS]: I wonder why this changes ? (seems unrelated to the PR)
Also bring a question on, shouldn't the others use the same (instead of copying the list on each file) ?

Comment (arilivigni) [reply ci-operator/config/openshift/tektoncd-pipeline/OWNERS]: I am following what other teams did and there is no OWNERS_ALIAS but I can add it

**Comment by vdemeester** [comment]
@arilivigni I think you can remove the `WIP` :angel:

**Comment by arilivigni** [comment]
@petr-muller can I get approval for this please

File: ci-operator/config/openshift/tektoncd-pipeline-operator/openshift-tektoncd-pipeline-operator-master.yaml [hunk 3]
```diff
@@ -0,0 +1,72 @@
+canonical_go_repository: github.com/openshift/tektoncd-pipeline-operator
+
+base_images:
+  os:
+    cluster: https://api.ci.openshift.org
+    name: centos
+    namespace: openshift
+    tag: '7'
+  operator-registry:
+    name: "4.0"
```
Comment (petr-muller) [inline ci-operator/config/openshift/tektoncd-pipeline-operator/openshift-tektoncd-pipeline-operator-master.yaml]: You're sure you build on OCP 4.0?

File: ci-operator/config/openshift/tektoncd-pipeline-operator/openshift-tektoncd-pipeline-operator-master.yaml [hunk 4]
```diff
@@ -0,0 +1,72 @@
+canonical_go_repository: github.com/openshift/tektoncd-pipeline-operator
+
+base_images:
+  os:
+    cluster: https://api.ci.openshift.org
```
Comment (petr-muller) [inline ci-operator/config/openshift/tektoncd-pipeline-operator/openshift-tektoncd-pipeline-operator-master.yaml]: Remove the `cluster:` part, it's not necessary and makes ci-operator treat the source as an external cluster.

File: ci-operator/config/openshift/tektoncd-pipeline-operator/openshift-tektoncd-pipeline-operator-master.yaml [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+  name: tektoncd-pipeline
+
+build_root:
+  project_image:
+    dockerfile_path: openshift-ci/Dockerfile.tools
+
+binary_build_commands: make build
+
+tag_specification:
+  name: "4.0"
```
Comment (petr-muller) [inline ci-operator/config/openshift/tektoncd-pipeline-operator/openshift-tektoncd-pipeline-operator-master.yaml]: You're sure you build on OCP 4.0?

File: ci-operator/config/openshift/tektoncd-pipeline-operator/openshift-tektoncd-pipeline-operator-master.yaml (hunk 4 above)
Comment (arilivigni) [reply ci-operator/config/openshift/tektoncd-pipeline-operator/openshift-tektoncd-pipeline-operator-master.yaml]: @petr-muller in promotion too?

File: ci-operator/config/openshift/tektoncd-pipeline-operator/openshift-tektoncd-pipeline-operator-master.yaml (hunk 3 above)
Comment (arilivigni) [reply ci-operator/config/openshift/tektoncd-pipeline-operator/openshift-tektoncd-pipeline-operator-master.yaml]: 4.1 then?

**Comment by chmouel** [comment]
This PR should have not been merged, nothing has been working on the operator repository side,
